### ISSUE TITLE
Plugin could not be registered at 'html-webpack-plugin-before-html-processing'. Hook was not found.

### ISSUE THREAD

**Issue by wind4gis** [GoogleChromeLabs/preload-webpack-plugin#86]
I'm using CRA which using `"html-webpack-plugin": "4.0.0-alpha.2"`
When I write it on my package.json
![snipaste_2018-12-26_16-11-20](https://user-images.githubusercontent.com/3963291/50438475-0f820880-0929-11e9-93a9-c90fbee03c5a.png)
However, it shows error
![snipaste_2018-12-26_16-12-57](https://user-images.githubusercontent.com/3963291/50438488-245e9c00-0929-11e9-9b98-bcac68be8725.png)

is the plugin don't support webpack4???

**Comment by rappad** [comment]
Yes the current version of this package does not work with `html-webpack-plugin v4`
Please refer this [issue](https://github.com/GoogleChromeLabs/preload-webpack-plugin/issues/79) 

or

you can try `yarn add preload-webpack-plugin@3.0.0-beta.3`

**Comment by Tim-ls** [comment]
I have same error.

**Comment by guirip** [comment]
Same error for me.

**Comment by indusoni** [comment]
@guirip 
Yes with preload-webpack-plugin@3.0.0-beta.3 ,its building successfully,but its not adding perload/prefetch .
config for this is  like below mentioned
  new PreloadWebpackPlugin({
        rel: 'preload',
        include:['home','vendor'] // or 'initial'
      }),

Webpack version : 4.28.3

**Comment by leifj** [comment]
fwiw 3.0.0-beta.3 worked for me and generated links

**Comment by eugenmihailescu** [comment]
Yes, 3.0.0-beta.3 worked for me too

**Comment by iampava** [comment]
Interesting... It didn't work for me

**Comment by tmladek** [comment]
Confirming, same error, specifying `3.0.0-beta.3` fixed the issue.

Looking forward to when it's not a beta! (:
